joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -712,6 +711,21 @@ bool JointTrajectoryController::validate_trajectory_msg(
     return false;
   }
 
+  const auto trajectory_start_time = static_cast<rclcpp::Time>(trajectory.header.stamp);
+  if (trajectory_start_time.seconds() != 0.0) {
```

Comment:
```suggestion
  // If the starting time it set to 0.0, it means the controller should start it now. 
  // Otherwise we check if the trajectory ends before the current time, in which case it can be ignored.
  if (trajectory_start_time.seconds() != 0.0) {
```

Reply:
I think a bit of a reminder is good to have here